test/spec/spectest.cpp

```diff
@@ -562,18 +569,27 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
   };
 
   // Command processing. Return true for expected result.
-  auto RunCommand = [&](const rapidjson::Value &Cmd) {
-    // Line number in wast: Cmd["line"].Get<uint32_t>()
-    if (const auto Type = Cmd.FindMember("type"s); Type != Cmd.MemberEnd()) {
-      switch (resolveCommand(Type->value.Get<std::string>())) {
+  auto RunCommand = [&](const simdjson::dom::object &Cmd) {
+    std::string_view TypeField;
+
+    if (!Cmd["type"].get(TypeField)) {
+      switch (resolveCommand(TypeField)) {
       case SpecTest::CommandID::Module: {
-        const auto FileName = (TestsuiteRoot / Proposal / UnitName /
-                               Cmd["filename"].Get<std::string>())
-                                  .u8string();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
-        if (const auto Name = Cmd.FindMember("name"); Name != Cmd.MemberEnd()) {
+        std::string_view Name = Cmd["filename"];
+        const auto FileName =
+            (TestsuiteRoot / Proposal / UnitName / Name).u8string();
+        const uint64_t LineNumber = Cmd["line"];
+        std::string LineStr = std::to_string(LineNumber);
+        std::string_view TempName;
+        if (!Cmd["name"].get(TempName)) {
           // Module has name. Register module with module name.
-          LastModName = Name->value.Get<std::string>();
+          if (auto It = Alias.find(std::string(TempName)); It != Alias.end()) {
```

Comment:
Why do we have the logic of `Alias` here? It seems like the original codes don't have this logic. Is the original code wrong?

Reply:
As I mentioned above, SIMDJSON doesn't allow us to modify JSON objects. In the previous code, the module names were added or modified in the JSON objects itself. Since I couldn't do it here, I used `Alias` to replace the Module names.